### ISSUE TITLE
Proposal: Automatic XML Validation 

### ISSUE THREAD

**Issue by nkprasad12** [PerseusDL/lexica#74]
I noticed that one of my recent PRs had a missing <sense> tag that wasn't caught until after it was merged, and needed to be fixed manually by lcerrato (https://github.com/PerseusDL/lexica/pull/71).

GitHub offers (for free, to public repos such as this one) the ability to run a suite of tests based on certain trigger criteria, reducing the burden on repo maintainers. 

If this seems useful, I can send a PR to automatically check that the `lat.ls.perseus-eng2.xml` file can be parsed as valid XML before any pull request can be merged. This check is non-binding, repo admins would still have the discretion to merge a PR that failed the checks if desired.

**Comment by lcerrato** [comment]
@nkprasad12 
We have this for other repos. We haven't implemented it here as this is not a part of our release packages and the text is not compliant with our other testing schemes.

**Comment by nkprasad12** [comment]
Thanks for the context!

As a final quick comment - even if not compliant with other testing schemes or part of the release packages, for internal consistency it looks to me like the repo would benefit from adding a basic XML validation test, and with with not so much overall cost (as the check can be easily removed if needed).

However, I of course understand the hesitance to add extra complication / potentially untrusted code to the repo. 

Please feel free to close.

**Comment by lcerrato** [comment]
@nkprasad12 
You are right. We had testing via an external provider in the pre-GitHub action setup. 
It's just been orphaned since then as nothing in this repo has any impact on any current releases. 

Obviously, we want to keep the xml well-formed for distribution purposes.
I should be able to spot anything in the course of my work (since I'm the main editor), but ideally there would be testing.
